microk8s-resources/actions/common/utils.sh

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+exit_if_stopped() {
+  # test if the snap is marked as stopped
+  if [ -e ${SNAP_DATA}/var/lock/stopped.lock ]
+  then
+    echo "microk8s is not running, try microk8s.start"
+    exit 1
+  fi
+}
```

Comment:
@ktsakalozos while we're in here, what do you think about trying to address a bit of #174 in this function?  I was thinking we expand it like this:
```bash
if -e stopped.lock
    echo "microk8s is not running, try microk8s.start"
    exit 1
else
  if check_apiserver: ## apiserver port unreachable; systemclt status apiserver != running; apiserver process not running; etc:
    while retry < 5
      echo "waiting for microk8s api server to start"
      sleep 2
      if check_apiserver == good
        return
      else
        retry++
    done
    echo "microk8s api server did not start, see journalctl"
    exit 1
  fi
fi
```
Since this PR adds the `exit_if_stopped` check to all the relevant wrappers, i think it makes sense to use this util as a "stopped or not running" gate.

Reply:
I am already on it @kwmonroe but not in this PR. Have a look here https://github.com/ubuntu/microk8s/pull/178/files#diff-0f01327987334a699c82fa97496f4895R10 . With the `microk8s.status` command you will be able to get the status (running or not), and also wait for microk8s to get ready (with a timeout or wait forever). :)